src/microbe_stage/MicrobeHUD.cs

```diff
@@ -246,7 +247,7 @@ public class MicrobeHUD : Control
     private ProgressBar oxygenBar = null!;
     private ProgressBar co2Bar = null!;
     private ProgressBar nitrogenBar = null!;
-    private ProgressBar temperature = null!;
+    private ProgressBar temperatureLabel = null!;
```

Comment:
This is of type ProgressBar, why did you add the Label suffix? That should only be used for things that are actual labels...